arbitrum-docs/bold/bold-gentle-introduction.md

```diff
@@ -0,0 +1,139 @@
+
+BOLD enables permissionless validation by allowing *anyone* to challenge incorrect Arbitrum state assertions and therefore unlocks new avenues for participation in securing the network, fostering greater inclusivity and resilience. This is made possible because BOLD guarantees that a single, honest entity staked on the correct Arbitrum state assertion will always win against malicious adversaries. The research and work to bring BOLD to life underscores Arbitrum's commitment to scaling Ethereum without compromising on security. 
+
+![pie-slice](./assets/l2beat-pie-chart.png)
+
+With BOLD at its core, Arbitrum charts a course towards being recognized as a Stage 2 rollup by addressing the currently yellow (above) State Validation wedge in [L2Beat's risk analysis pie chart](https://l2beat.com/scaling/summary). BOLD contributes to a more decentralized, efficient, and robust rollup ecosystem. Additionally, BOLD will be available as an upgrade to all Orbit chains who wish to adopt it to reap the aforementioned benefits.
+
+### How is this possible?
+The BOLD protocol provides the guardrails and rules for how validators produce, validate, and challenge claims about the state of an Arbitrum chain. Since Arbitrum’s state is deterministic, there will always be only 1 correct state for a given input of on-chain operations and transactions. The beauty of BOLD’s design is centered around this very fact: that there is only 1 correct state on Arbitrum and therefore anyone defending that correct state will always prevail.
+
+Let’s dive in to an overview of how BOLD actually works.
+
+1. **An assertion is made:** Validators begin by taking the most recent confirmed L2 block, called `Block A`, and assert that some number of transactions afterwards, using Nitro’s deterministic State Transition Function (STF), will result in an end state, `Block Z`. If a validator claims that the end state represented by `Block Z` is correct, they will stake on `Block Z` and propose that state to be posted to Ethereum. If nobody disagrees after a certain amount of time, known as the challenge period, then the state represented by the L2 `Block Z` is confirmed as the correct state of an Arbitrum chain. However, if someone disagrees with the end state `Block Z`, they can submit a challenge. This is where BOLD comes in to play.
```

Comment:
It doesn't work exactly like this. There's general some misconception around what are the differences between L2 blocks and Rollup blocks (or RBlocks). There's always been that misconception though 😄 

In the challenge protocol, RBlocks are used, which are unrelated to L2 blocks. There's more info in [this FAQ](https://docs.arbitrum.io/learn-more/faq#what-is-the-difference-between-an-l2-block-and-a-rblock) and [this page](https://docs.arbitrum.io/inside-arbitrum-nitro/#arbitrum-rollup-protocol)